board/safety/safety_honda.h

```diff
@@ -144,14 +162,24 @@ static int honda_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
     }
   }
 
-  // GAS: safety check
+  // GAS: safety check (nidec)
   if (addr == 0x200) {
     if (!current_controls_allowed || !long_controls_allowed) {
       if (GET_BYTE(to_send, 0) || GET_BYTE(to_send, 1)) {
         tx = 0;
       }
     }
   }
+  // GAS: safety check (bosch)
+  if (addr == 0x1DF) {
```

Comment:
can be merged with brake check. Is the same message

Reply:
fixed